### PR TITLE
ES-829615-Add-Subpage-for-Azure-App-Service-WordtoPDF

### PR DESCRIPTION
**Pull request by Venkateshmuruganandam**
Hi All,
Added Subpages of Azure App service for Convert Word to PDF.

Regards
Venkatesh

### CODE REVIEW

File: File-Formats-toc.html [hunk 1]
```diff
@@ -249,7 +249,24 @@
 							</li>
 							<li>
 								<a href="/file-formats/DocIO/Convert-Word-Document-to-PDF-in-MAC">Mac</a>
-							</li>		
+							</li>
+							<li>
+								<a href="/file-formats/DocIO/Convert-Word-Document-to-PDF-in-Azure">Azure</a>
+								<ul>
+									<li>
+										<a href="/file-formats/DocIO/Convert-Word-Document-to-PDF-in-Azure-Functions-v1">Azure Function v1</a>
```
Comment (MohanaselvamJothi) [inline File-Formats-toc.html:257]: It should be Azure Functions not Function

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-Azure-App-Service-Windows.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+
+using Syncfusion.DocIO;
+using Syncfusion.DocIO.DLS;
+using Syncfusion.DocIORenderer;
+using Syncfusion.Pdf;
+using Microsoft.AspNetCore.Hosting;
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Convert-Word-Document-to-PDF-in-Azure-App-Service-Windows.md:79]: Remove the unwanted namespace

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-Azure-App-Service-Windows.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+        string extension = Path.GetExtension(Request.Form.Files[0].FileName).ToLower();
+        // Compares extension with supported extensions.
+        if (extension == ".docx")
+        {
+            MemoryStream stream = new MemoryStream();
+            Request.Form.Files[0].CopyTo(stream);
+            try
+            {
+                //Open using Syncfusion
+                using (WordDocument document = new WordDocument(stream, Syncfusion.DocIO.FormatType.Docx))
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Convert-Word-Document-to-PDF-in-Azure-App-Service-Windows.md:125]: Why we need to mention full naming Syncfusion.DocIO.FormatType.Docx?

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-Azure-App-Service-Windows.md [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+![Publish has been succeeded](Azure_Images/App_Service_Linux/After_Publish_WordtoPDF.png)
+
+Step 11: Now, the published webpage will open in the browser. 
+![Browser will open after publish](Azure_Images/App_Service_Linux/Browser_WordtoPDF.png)
+
+Step 12: Select the Word document and Click **Convert to PDF** to convert the given Word document to a PDF.You will get the output PDF document as follows.
+
+![Output image PDF document](WordToPDF_images/WordToPDF_Output_Cloud.png)
+
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Word-to-PDF-Conversion/Convert-Word-document-to-PDF/Azure/Azure_Functions/Azure_Functions_v1).
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Convert-Word-Document-to-PDF-in-Azure-App-Service-Windows.md:207]: Add valid hyperlink

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats-toc.html [hunk 5]
```diff
@@ -249,7 +249,24 @@
...
+									</li>
+									<li>
+										<a href="/file-formats/DocIO/Convert-Word-Document-to-PDF-in-Azure-Functions-v4">Azure Function v4</a>
+									</li>
+									<li>
+										<a href="/file-formats/DocIO/Convert-Word-Document-to-PDF-in-Azure-App-Service-Linux">Azure App Service (Linux)</a>
+									</li>
+									<li>
+										<a href="/file-formats/DocIO/Convert-Word-Document-to-PDF-in-Azure-App-Service-Windows">Azure App Service (Windows)</a>
```
[the remark this answers is not in the document]
Comment (Venkateshmuruganandam) [reply File-Formats-toc.html:266]: Updated

File: File-Formats-toc.html (hunk 1 above)
Comment (Venkateshmuruganandam) [reply File-Formats-toc.html:257]: Updated

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-Azure-App-Service-Windows.md (hunk 2 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-PDF-in-Azure-App-Service-Windows.md:79]: updated

File: File-Formats/DocIO/Convert-Word-Document-to-PDF-in-Azure-App-Service-Windows.md (hunk 3 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-PDF-in-Azure-App-Service-Windows.md:125]: Updated

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**